src/Screens/Gameplay/Components/Map/FocusedMap/FocusedMap.jsx

```diff
@@ -0,0 +1,32 @@
+import './FocusedMap.css';
+import Tile from './Tile.jsx';
+
+const GRID_ROWS = 15;
+const GRID_COLS = 26;
+
+const newNode = Tile;
+
+const grid = [];
+for (let row = 0; row < GRID_ROWS; row++) {
+	const currentRow = [];
+	for (let col = 0; col < GRID_COLS; col++) {
+		currentRow.push(newNode);
```

Comment:
this array shouln't store components, but data objects recieved from DB - 
![image](https://user-images.githubusercontent.com/67635802/116807683-26f6df00-ab3d-11eb-8d33-650266c9285a.png)